### ISSUE TITLE
Add Hash of data to the macaroon caveat

### ISSUE THREAD

**Issue by avemeva** [bitlum/macaroon-application-auth#1]
If we not hash the request data, than attacker could intercept the token and just change the request data.

**Comment by talionarwork** [comment]
Do you mean some one can intercept token from integrations like ZAP? Because for internal communication we have Private Networking on DO that is not connected to the Internet at all

**Comment by AndrewSamokhvalov** [comment]
Nah, now it used only inside our infrastructure, with Zap we not gonna use any auth tokens.

**Comment by talionarwork** [comment]
Then we can skip this, because nobody can intercept Private Network. Only one way to make this if they will have access by SSH to our droplets. But in that case they will control everything so I afraid that just hash can't help us anyway
